recipes/dictobj/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "dictobj" %}
+{% set version = "0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 15d6ac1c720350dcce3d01c31882cbc8e4a14cb22a8bca290a18ca7b0c0988f1
+
+build:
+  number: 0
```

Comment:
Any reason not to?
```suggestion
  noarch: python
  number: 0
```

Reply:
No reason, just an oversight ... Thanks for the catch!